ceph_salt/deploy.py

```diff
@@ -1242,7 +1242,7 @@ def check_deploy_prerequesites(minion_id):
         host_ls = []
         for minion, value in result.items():
             if value:
-                host_ls = SaltClient.local().cmd(minion, 'ceph_orch.host_ls')[minion]
+                host_ls = SaltClient.local_cmd(minion, 'ceph_orch.host_ls').get(minion, [])
```

Comment:
I don't see any situation where we need `.get(minion, [])` instead of `[minion]`. Are you changing this for any particular reason?

If we are calling commands in a minion and that minion is not part of the salt result dict, then we should not ignore that by assuming the `result` is `[]`. 

I'm in favor of keeping the `[minion]`.

Reply:
> I don't see any situation where we need `.get(minion, [])` instead of `[minion]`. Are you changing this for any particular reason?

Indeed,when fixing the original issue I had, the minion was indeed missing form this dict!

> If we are calling commands in a minion and that minion is not part of the salt result dict, then we should not ignore that by assuming the `result` is `[]`.
> 
> I'm in favor of keeping the `[minion]` instead of `.get(minion, [])`.

What about verify this in `SaltClient.local_cmd`?

Reply:
> What about verify this in `SaltClient.local_cmd`?

Agree. And raise an exception if that happen.

Reply:
done!